tests/system/test_root_metronome.py

```diff
@@ -281,8 +281,7 @@ def test_metronome_shutdown_with_no_extra_tasks():
         # restart metronome process
```

Comment:
we should drop this comment

Reply:
to avoid conflict and duplication I rebased this on the other PR... removing.